src/components/BorrowerProfile/RepaymentSchedule.vue

```diff
@@ -26,7 +26,53 @@
 						{{ repaymentStatusCheck }}.
 					</p>
 				</span>
-				<table class="tw-table-auto">
+
+				<!-- Table for small screens -->
+				<table class="md:tw-hidden tw-w-full">
+					<tr
+						v-for="(repayment, index) in parsedRepaymentSchedule"
+						:key="index"
+						class="tw-mb-1"
+					>
+						<div class="tw-bg-secondary tw-rounded tw-text-center tw-mb-2 tw-pb-1.5">
+							<p class="tw-text-h4 tw-py-1.5">
+								{{ repayment.formattedRepaymentDate }}
+							</p>
+							<hr class="tw-mb-1.5 tw-mx-1.5">
+							<p class="tw-mb-1.5">
+								Expected: {{ repayment.formattedMonthlyPayment }}
+							</p>
+							<p v-if="!repayment.repaid && !repayment.delinquent">
+								Available {{ repayment.formattedRepaymentDate }}
+							</p>
+							<!-- if payment is received -->
+							<p
+								class="tw-bg-primary tw-mx-auto tw-w-[11.5rem] tw-py-1 tw-rounded"
```

Comment:
Ideally try to avoid these bracketed `tw-w-[555rem]` classes because they end up going into the stylesheet that is loaded on all pages even though they're probably just used on this page.  Over time the stylesheet will get bloated.

For these, adding a `style="width: 11.5rem;"` would work well.

The main reason I've seen to use the bracketed styles is when you have to change things responsively like
`tw-w-[11.5rem] lg:tw-w-[12rem]`. Since you can't do media queries with inline styles.

Reply:
Thanks for the explanation @ryan-ludwig